pkg/framework/aries/api/vdri/vdri.go

```diff
@@ -118,6 +119,13 @@ func WithServiceEndpoint(serviceEndpoint string) DocOpts {
 	}
 }
 
+// WithRoutingKeys allows for setting routing keys.
+func WithRoutingKeys(routingKeys []string) DocOpts {
```

Comment:
This will need to be revisited for multiple router support. Also there is the option of not using a router (even when routers are configured).

Reply:
The use case of not using a router is fulfilled by not passing this option when creating the DID.

Reply:
Added this to the list #1076 .